packages/orbit-components/src/Loading/index.tsx

```diff
@@ -29,7 +29,9 @@ const Loader = ({ type, customSize }) => {
         viewBox={`0 0 ${customSize} ${customSize}`}
         className="orbit-loading-spinner animate-spinner"
         style={{ height: `${customSize}px`, width: `${customSize}px` }}
+        role="img"
       >
+        <title>{title}</title>
```

Comment:
Added `title` element based on the article from [Deque blog](https://www.deque.com/blog/creating-accessible-svgs/) (pattern 5).

Reply:
It is arguable if we need this. The loading SVG is not exactly an image like the use cases presented in that blog page, it is more of an animation that represents a state. For that, other aria attributes can be used. But I am ok with moving forward with this one and adjust later if needed.